app/models/mixins/authentication_mixin.rb

```diff
@@ -130,7 +130,7 @@ def update_authentication(data, options = {})
     data.each_pair do |type, value|
       cred = authentication_type(type)
       current = {:new => nil, :old => nil}
-
+      
```

Comment:
I've managed to get my local rubocop linter working in Atom. So I'll be able to catch these better next time. Sorry 🍰